src/validators/target/index.js

```diff
@@ -1,7 +1,20 @@
-import {noop} from 'lodash'
+const validValues = [
+  'web', 'webworker', 'node', 'async-node', 'node-webkit', 'electron',
+]
+const validValuesList = validValues.join(', ')
+
 export default {
   key: 'target',
-  validate: noop,
+  validate: validateTarget,
 }
 
-
+function validateTarget(value) {
+  const type = typeof value
+  if (type !== 'string') {
+    return `Unexpected type ${type}. Must be a string of one of: ${validValuesList}`
+  }
+  const isValid = validValues.indexOf(value) !== -1
```

Comment:
Do we need the `const isValid` or can we push `validValues.indexOf(value) !== -1` for the condition in the next `if` statements?